VisualPinball.Unity/VisualPinball.Unity.Editor/Utils/TreeViewWithTreeModel/TreeModel.cs

```diff
@@ -0,0 +1,232 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace VisualPinball.Unity.Editor.Utils.TreeViewWithTreeModel
```

Comment:
Is this layer-agnostic?

I would prefer a shorter name. How about `VisualPinball.Unity.Editor.Utils.TreeView`?

Reply:
yup it's a base framework to easily populate a TreeModel of TreeElements into a special TreeView called TreeViewWithTreeModel, the TreeView base class is not as easy to setup.